test/ServerComparison.FunctionalTests/NoCompression.config

```diff
@@ -177,7 +177,9 @@
 
     <system.webServer>
 
-        <serverRuntime />
+      <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" hostingModel="[HostingModel]" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
```

Comment:
Can we keep it in web.config and let hosting merge them,?

Reply:
No, I need to delete the web.config so Express does not try to use it. It will be a closer match to the VS scenario where there is no web.config by default.

Reply:
Are you referring to the samples in the repository.

Reply:
Was confused myself before. This is fine.